grafana_wtf/core.py

```diff
@@ -321,28 +321,32 @@ def info(self):
         return response
 
     @property
-    def health(self):
+    def build_info(self):
         response = None
         error = None
-        error_template = f"The request to {self.grafana_url.rstrip('/')}/api/health failed"
+        error_template = f"The request to {self.grafana_url.rstrip('/')}/api/frontend/settings failed"
         try:
-            response = self.grafana.client.GET("/health")
+            response = self.grafana.client.GET("/frontend/settings")
             if not isinstance(response, dict):
                 error = f"{error_template}: Invalid response, content was: {response}"
 
+            response = Munch(response)
+            response = response.get("buildInfo")
```

Comment:
The whole response is quite large. It makes sense to return only basic `buildInfo` which contains similar to `/api/health` response data.